frontstage/views/surveys/surveys_list.py

```diff
@@ -14,7 +14,7 @@
 logger = wrap_logger(logging.getLogger(__name__))
 
 
-@surveys_bp.route('/', methods=['GET'])
+@surveys_bp.route('/todo', methods=['GET'])
```

Comment:
We can have this at `/todo` but we also want `/` to redirect here I think when the user is signed in? I think when a user signs in they are redirected to `/` so that would need to be updated as well

Reply:
When I sign in I get redirected to `/todo` but I'll have a look see 👍

Reply:
So it redirects to `url_for('surveys_bp.logged_in')` which is this route, not named the best, but works.